packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -203,10 +241,15 @@ export function drawLivenessOvalInCanvas({
   const ctx = canvas.getContext('2d');
 
   if (ctx) {
+    ctx.restore();
     ctx.clearRect(0, 0, canvasWidth, canvasHeight);
 
     // fill the canvas with a transparent rectangle
-    ctx.fillStyle = 'rgba(255, 255, 255, 1.0)';
+    ctx.fillStyle = getComputedStyle(canvas).getPropertyValue(
+      isStartScreen
+        ? '--amplify-colors-background-primary'
+        : '--amplify-colors-white'
```

Comment:
Think this reads like it should be evaluated in a ui component. Does it need to be here?

Reply:
Not necessarily, just added it here because I didn't want to do this style computation at the caller of drawLivenessOvalInCanvas which happens in a few places now

Reply:
Could we put it in a function we use at the call site instead?